docs/src/domains/steps/_index.md

```diff
@@ -3,75 +3,122 @@ title: "Custom Domains - Step by step guide"
-even if your project is on a Production plan.
-{{< /note >}}
+You need to:
 
-In the [console](https://console.platform.sh), click the **More** button for your project and select **Edit plan**.
+- Have a project that’s ready to be live.
+- Have a domain and have access to its settings on the registrar’s website.
+- (Optional) Have the [CLI](../../development/cli/_index.md) installed locally.
+- Make sure that your registrar allows [CNAMEs on Apex domains or one of the alternatives](../steps/dns.md).
 
-![Edit Plan](/images/management-console/edit-plan.png "0.3")
+## 1. Change your plan to a production plan
 
-You can also access information about the project's plan under "Billing"
-and then by selecting the project from your list of projects.
-You can make changes to the project by clicking ‘Upgrade Plan’.
+Note that the domain used for non-production environments is always generated and can't be customized,
+even if your project is on a Production plan.
 
-![Edit Plan](/images/management-console/billing-plan-upgrade.png "0.6")
+If you are on a Development plan, you can't add a domain.
+To upgrade your subscription to a production plan:
 
-You can make changes to the type of plan, the number of environments, amount of storage and number of users here.
-When you make changes, it will update the monthly price you will be paying. Click `Upgrade plan` to save the new settings.
+<!--This is in HTML to get the icon not to break the list. -->
+<ol>
+  <li>Access the <a href="https://console.platform.sh">project overview</a></li>
+  <li>On the tile of the project you want to edit, click {{< icon more >}}</strong>More</strong>.</li>
+  <li>Click <strong>Edit plan</strong>.</li>
+  <li>Change the type of plan to at least <code>Standard</code> to go live. When you make changes, the monthly price you pay is updated.</li>
+  <li>Click <strong>Save</strong>.</li>
+</ol>
 
-![Edit Plan Choose](/images/management-console/settings-upgrade-plan.png "0.6")
+TODO update screenshots and delete the legacy ones
+<!-- ![Edit Plan](/images/management-console/edit-plan.png "0.3")
+![Edit Plan Choose](/images/management-console/settings-upgrade-plan.png "0.4") -->
 
 You can find more information on pricing on the [pricing page](https://platform.sh/pricing).
 
-## 2. (CDN version) Configure your DNS provider
+## 2. Configure your DNS provider
+
+{{< codetabs >}}
+
+---
+title=With a CDN
+file=none
+highlight=false
+---
 
 If you are serving the site through a CDN, configure your DNS provider to point at your CDN account.
-The address or CNAME to set for that will vary with the CDN provider.
+The address or CNAME to set for that varies with the CDN provider.
 Refer to their documentation or to the [CDN guide](/domains/cdn/_index.md).
 
-## 2. (Non-CDN version) Configure your DNS provider
+<--->
 
-Configure your DNS provider to point your domain to your Platform.sh production environment domain name.
+---
+title=Without a CDN
```

Comment:
:hammer_and_wrench: I'd have this tab include step 3 (so `With a CDN` wouldn't have a step 2. That might make it clearer what's necessary with a CDN and without. Though not sure how they appear in the ToC that way :thinking: